src/lib/parameters/CMakeLists.txt

```diff
@@ -82,6 +82,14 @@ endif()
 if(CONFIG_BOARD_IO)
 	set(board_with_io_arg --board-with-io)
 endif()
+
+# TODO: Why do these only need to be filtered out on posix?
+if (${PX4_PLATFORM} STREQUAL "posix")
```

Comment:
What was going on here?

Reply:
Hmmm weird

Reply:
Alright, I fixed this one and pushed up the change. It was ```set(DISABLE_PARAMS_MODULE_SCOPING TRUE)``` causing the problem.